jubilant/_juju.py

```diff
@@ -205,6 +205,151 @@ def add_unit(
 
         self.cli(*args)
 
+    def bootstrap(
+        self,
+        cloud: str | None = None,
+        controller: str | None = None,
+        *,
+        add_model: str | None = None,
+        agent_version: str | None = None,
+        auto_upgrade: bool = False,
+        bootstrap_base: str | None = None,
+        bootstrap_constraints: Mapping[str, str] | None = None,
+        bootstrap_image: str | None = None,
+        bootstrap_series: str | None = None,
+        build_agent: bool = False,
+        clouds: bool = False,
+        config: Mapping[str, ConfigValue] | None = None,
+        constraints: Mapping[str, str] | None = None,
+        controller_charm_channel: str | None = None,
+        controller_charm_path: str | pathlib.Path | None = None,
+        credential: str | None = None,
+        db_snap: str | pathlib.Path | None = None,
+        db_snap_asserts: str | pathlib.Path | None = None,
+        force: bool = False,
+        keep_broken: bool = False,
+        metadata_source: str | pathlib.Path | None = None,
+        model_default: Mapping[str, ConfigValue] | None = None,
+        no_switch: bool = False,
+        regions: str | None = None,
+        storage_pool: Mapping[str, str] | None = None,
+        to: str | None = None,
```

Comment:
This should probably be `str | Iterable[str] | None` like `deploy` and `add_unit`.